### PR TITLE
README update to include working example

### PR DESCRIPTION
**Pull request by antdjohns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,14 +43,13 @@ c := polygon.NewWithClient("YOUR_API_KEY", hc)
 After creating the client, making calls to the Polygon API is simple.
 
 ```golang
-params := models.GetAllTickersSnapshotParams{
-    Locale:     models.US,
-    MarketType: models.Stocks,
-}.WithTickers("AAPL,MSFT")
+params := models.GetTickerDetailsParams{
+Ticker: "AAPL",
+}
```
Comment (morningvera) [inline README.md:48]: can we add `WithDate(...)` to the example so that we're still demonstrating how to use query params?

Comment (antdjohns) [reply README.md:48]: ty

File: README.md [hunk 2]
```diff
@@ -43,14 +43,13 @@ c := polygon.NewWithClient("YOUR_API_KEY", hc)
...
 
-res, err := c.GetAllTickersSnapshot(context.Background(), params)
+res, err := c.GetTickerDetails(context.Background(), &params)
 if err != nil {
-    log.Fatal(err)
+log.Fatal(err)
```
[the remark this answers is not in the document]
Comment (antdjohns) [reply README.md:52]: got it.

File: README.md [hunk 3]
```diff
@@ -43,12 +43,10 @@ c := polygon.NewWithClient("YOUR_API_KEY", hc)
 After creating the client, making calls to the Polygon API is simple.
 
 ```golang
-params := models.GetAllTickersSnapshotParams{
-    Locale:     models.US,
-    MarketType: models.Stocks,
-}.WithTickers("AAPL,MSFT")
+params := models.GetTickerDetailsParams{Ticker: "AAPL"}.
+	WithDate(models.Date(time.Date(2021, 7, 22, 0, 0, 0, 0, time.Local)))
```
Comment (morningvera) [inline README.md:47]:
```suggestion
params := models.GetTickerDetailsParams{
    Ticker: "AAPL",
}.WithDate(models.Date(time.Date(2021, 7, 22, 0, 0, 0, 0, time.Local)))
```

**Review by morningvera — APPROVED** [review]
lgtm

**Pull request merged**